packages/core/src/extensions/LinkToolbar/LinkToolbarPlugin.ts

```diff
@@ -190,7 +190,7 @@ class LinkToolbarView implements PluginView {
     }
   }
 
-  update(view: EditorView, oldState?: EditorState) {
+  update(view: EditorView, oldState?: EditorState, fromMouseOver = false) {
```

Comment:
Am I crazy to think that no one else is calling this update method?

Reply:
Ah this is part of the `PluginView` spec from ProseMirror, it automatically gets called on state updates (https://prosemirror.net/docs/ref/#state.PluginView). It's just harder to see that since in our case the view does a lot of stuff.

Reply:
Also I guess normally you'd pass in a plain object, so the fact that we're using a class also obfuscates things

Reply:
Ohh ok. Eh, this definitely is a hack. But, I think it is okay for now